CodeforAustralia/council-emissions-calculator
# Download results

## Overview of the Pull Request:
Create the following component design for downloading PDF and CSV files:
![image](https://user-images.githubusercontent.com/88268603/170807659-1e09c4b2-15e2-42ff-920b-83e90896c23f.png)

**Out of scope:**
Display "absolute" didn't quite work with Chakra UI elements. Will re-visit this if we have time.
![image](https://user-images.githubusercontent.com/88268603/170680893-e44cc92a-24ef-4464-a844-79956eab7540.png)

## link to Trello card or Github issue
https://trello.com/c/rrgdNx26

## How Has This Been Tested?
- Load the code. 
- Resize the page to check responsiveness.
- Check hover effects for the buttons.

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!
